### DISCUSSION TITLE
post/2021/2021-09-pedantic-clippy/

### DISCUSSION THREAD

**Discussion by giscus** [zyrikby/website_related#24] [Announcements]
# post/2021/2021-09-pedantic-clippy/

One of the reasons developers love Rust is its well-developed ecosystem. Clippy, a linter for the Rust code, is one of the main components in this ecosystem. It performs additional checks of the developed code reporting found issues and explaining how to fix them (and sometimes it even can fix them automatically). Its usage may be beneficial for Rust beginners and even professionals. In this article, I describe this tool and explain how to start using it.

https://zhauniarovich.com/post/2021/2021-09-pedantic-clippy/

**Comment by YoungHaKim7** [comment]
thank you👍

**Comment by rtdietrich** [comment]
You saved my Life! :-)
